app/src/main/java/com/patrycja/filip/machinelearning/db/entity/ChapterProgressEntity.java

```diff
@@ -0,0 +1,37 @@
+package com.patrycja.filip.machinelearning.db.entity;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.ForeignKey;
+import android.arch.persistence.room.PrimaryKey;
+
+import java.util.Date;
+
+import lombok.Data;
+
+/**
+ * Created by zwsfke on 2017-11-26.
+ */
+@Data
+@Entity(tableName = "chapter_progress",
+        foreignKeys = {
+                @ForeignKey(entity = ChapterEntity.class,
+                        parentColumns = "id",
+                        childColumns = "chapter_id")
+        })
+public class ChapterProgressEntity {
+    @PrimaryKey(autoGenerate = true)
+    private int id;
+
+    @ColumnInfo(name = "chapter_id")
+    private int chapterId;
+
+    @ColumnInfo(name = "question_id")
+    private int questionId;
+
+    @ColumnInfo(name = "correct_answered")
+    private boolean correctAnswered;
```

Comment:
Jeśli chapter_progress ma przedstawiać przechodzenie kolejnych stron rozdziału to chapter sam w sobie nie jest pytaniem z mierzalnym czasem odpowiedzi, a raczej stroną do przeczytania/obejrzenia. Mierzalna będzie napewno czas odpowiedzi na quiz. Narazie niech zostanie zobaczymy co trzeba będzie zmienić jak zaczniemy implementować chapter.

Reply:
Nieźle!! o takie code review mi chodzi :) 

Kompletnie o tym zapomniałem, że będziemy mieć strony tylko z tekstem i takie gdzie będzie quiz...
Trzeba będzie to przerobić! :)